### DISCUSSION TITLE
Hosting docker with https

### DISCUSSION THREAD

**Discussion by AmitGarage** [milvus-io/milvus#33807] [Q&A and General discussion] [answered]
I hosted milvus docker locally on a remote Ubuntu server and it's working fine and able to connect from my pc. Now , I want to create it as a https url by adding certificate. Any suggestions how to do this?

**Comment by xiaofan-luan** [comment]
I think you just need to export 19530 port to outside then you can do restful api search on it

**Reply by AmitGarage** [reply]
Thanks for the reply. But , I already had exported port 19530

**Comment by AmitGarage** [comment, marked as answer]
I got it working. Had to do some changes in the milvus.yaml file and when again docker image is created . We can access milvus docker with https url.

Urls - 
https://milvus.io/docs/tls.md
https://milvus.io/docs/configure-docker.md
